add missing snippets (#352)
apostrophecms/a3-docs: docs/reference/modules/settings.md

        // This preview will be automatically generated.
        // preview: '{{ firstName }} {{ lastName }}'
      },

>       // The `adminLocale` option must be configured in the `@apostrophecms/i18n` module for this to be allowed

Reviewer: Oops!

>       // The `adminLocale` option must be configured in the `@apostrophecms/i18n` module for this to be allowed

Author: there was a typo also, it is `adminLocales` option

```

> <template v-slot:caption>

Reviewer: Doh!

  /modules/@apostrophecms/settings/index.js
</template>
</AposCodeBlock>

### Requirements

> All fields must be part of the `@apostrophecms/user` schema, except `adminLocale` that will be added automatically to

Reviewer: ```suggestion
All fields added to the subforms must be included as default fields or added to the `@apostrophecms/user` schema, except `adminLocale`. 
```

> `@apostrophecms/user` if the `adminLocales` option is set in the `@apostrophecms/i18n` module.

Reviewer: ```suggestion
If the `adminLocales` option is set in the `@apostrophecms/i18n` module, this field will be automatically added to the available subform schema fields and will add a select input for Admin language to the User management modal.
```

```

<template v-slot:caption>
  /modules/@apostrophecms/settings/index.js
</template>
</AposCodeBlock>

### Requirements

All fields must be part of the `@apostrophecms/user` schema, except `adminLocale` that will be added automatically to
`@apostrophecms/user` if the `adminLocales` option is set in the `@apostrophecms/i18n` module.

<AposCodeBlock>

``` javascript
module.exports = {
  fields: {
    add: {
      firstName: {
        type: 'string',
        label: 'First Name'
      },
      lastName: {
        type: 'string',
        label: 'Last Name'
      },
      displayName: {
        type: 'string',
        label: 'Display Name'
      }
    },
    group: {
      account: {
        label: 'Account',
        fields: [
          'firstName',
          'lastName',
          'displayName'
        ]
      }
    }
  }
};

```

<template v-slot:caption>
  /modules/@apostrophecms/user/index.js
</template>
</AposCodeBlock>

> <AposCodeBlock>

Reviewer: We could consider making this shorter, in case the user wants to copy-paste there is less to toss if it doesn't pertain to their project.
